content/flux/v0.x/stdlib/experimental/to.md

```diff
@@ -84,6 +89,10 @@ The organization ID of the specified [`bucket`](#bucket).
 Only required when writing to a different organization or a remote host.
 `orgID` and `org` are mutually exclusive.
 
+{{% warn %}}
+It is not possible to write to from one InfluxDB Cloud organization to another
```

Comment:
Technically you can write between organizations, you just can't with `experimental.to`.

```suggestion
`experimental.to()` cannot write to from one InfluxDB Cloud organization to another.
```